### PR TITLE
Prepare release v0.7.0

### PR DESCRIPTION
**Pull request by techwritermat**
Preparing release ~0.6.5~ 0.7.0 with Docusaurus 3.0 support.

### CODE REVIEW

File: .vscode/settings.json
Comment (bourdakos1) [inline .vscode/settings.json:1]: Can we drop this change?

**Review by bourdakos1 — CHANGES_REQUESTED** [review]
Since this is a breaking change, can we do 0.7.0 instead?

File: .vscode/settings.json
Comment (techwritermat) [reply .vscode/settings.json:1]: Sure thing

**Review by bourdakos1 — APPROVED** [review]

**Comment by bourdakos1** [comment]
Can you update the PR title to pass the title check and resolve the lint issues?

**Comment by techwritermat** [comment]
> Can you update the PR title to pass the title check and resolve the lint issues?

done boss

**Comment by bourdakos1** [comment]
Still looks like there’s a couple lint issues

**Comment by bourdakos1** [comment]
Not sure why these weren’t failing before, your PR doesn’t have any code changes :/

**Comment by techwritermat** [comment]
> Not sure why these weren’t failing before, your PR doesn’t have any code changes :/

Yeah, it's weird. I ran prettier on the three files the linter complained about. Hopefully, that fixes it 🤞

**Comment by axel7083** [comment]
Wonderful jobs !

**Pull request merged**